### ISSUE TITLE
Makefile includes

### ISSUE THREAD

**Issue by tj** [clibs/clib#121]
Just a thought but it might be cool to have some packages (or just one) with some Makefile targets that everyone can install/include to normalize things a bit. Stuff like clang-format, valgrind, blah blah

**Comment by jwerle** [comment]
yes! Perhaps a new attribute in the `clib.json` file to support a list of arbitrary files that cam be downloaded into the `deps/` directory
